src/stan_math_backend/Stan_math_code_gen.ml

```diff
@@ -509,7 +512,6 @@ let pp_write_array ppf {Program.prog_name; generate_quantities; _} =
     [ "using local_scalar_t__ = double;"; "vars__.resize(0);"
     ; "stan::io::reader<local_scalar_t__> in__(params_r__, params_i__);"
     ; strf "%a" pp_function__ (prog_name, "write_array")
-    ; strf "%a" pp_unused "function__"
```

Comment:
This can be removed because pp_function__ calls pp_unused internally. So this was just a duplicate.